src/tests/Microsoft.PowerFx.Interpreter.Tests/PADIntegrationTests.cs

```diff
@@ -121,6 +121,14 @@ public void DataTableEvalTest2()
         { Other : 5}
      )).Other");
             Assert.IsType<BlankValue>(result4);
+
+            var symbol = new SymbolTable();
+            symbol.EnableMutationFunctions();
+
+            engine.Config.SymbolTable = symbol;
+
+            var result5 = engine.Eval("Remove(robintable, {Names:\"name2\"});CountRows(robintable)", options: new ParserOptions() { AllowsSideEffects = true });
```

Comment:
This is perfect!

2 additional asserts:
- before this eval, we should assert table.CountRows == 3. 
- after this eval, we should assert table.CountRows == 2 (ie, that the actual table object is modified)